### PR TITLE
feat: Modify order schema

### LINKED ISSUE #466: Add new order schema and add migrations
Order API is almost ready and we should start adopting it.

- [ ] Add new order schema
- [ ] Add corresponding migration

Working on it

### PR DESCRIPTION
**Pull request by iamareebjamal**
- Add/Remove relevant attributes
- Increment DB version
- Add version migration
- Remove old references from layout

Fixes #466

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@mejariamol Please review

**Comment by iamareebjamal** [comment]
@mejariamol Please review

File: app/src/main/java/org/fossasia/openevent/app/common/data/models/Order.java [hunk 1]
```diff
@@ -10,20 +13,38 @@
 import lombok.Data;
 
 @Data
+@Type("ticket")
```
Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/common/data/models/Order.java]: why type is `ticket`? It should be `order` right?

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/common/data/models/Order.java]: Yeah fixing that :rofl:

**Comment by iamareebjamal** [comment]
@mejariamol Please review now

**Comment by iamareebjamal** [comment]
@mejariamol Please review now

**Pull request merged**